build/.vsts-ci.yml

```diff
@@ -392,7 +392,7 @@ jobs:
       IOT_DPS_INDIVIDUAL_X509_KEY: $(IOTHUB-E2E-X509-ECC-PRIVATE-KEY-BASE64)
       IOT_DPS_INDIVIDUAL_REGISTRATION_ID: $(IOT-DPS-INDIVIDUAL-REGISTRATION-ID)
   - task: PublishTestResults@2
-    displayName: 'Publish Ubuntu 20 Results'
+    displayName: 'Publish Ubuntu 20 (Debug) Results'
```

Comment:
No, ctest needs this or else it will fail looking for the release bits (stuff under release directory). This is why I had to add this. It was because of a pipeline break

Reply:
Sorry - I missed that you'd updated the test scripts here among the other changes and was looking at checked in behavior of those.